.github/workflows/build.yml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches:
       - main
+  merge_group:
```

Comment:
The file from the actions repo referenced below in the `build` job runs the 3 checks that the merge queue was expecting: build, codeql and git-secrets.

https://github.com/cloudscape-design/actions/blob/0d8c85cf4108447ca9625aaea94cf73e034fc0eb/.github/workflows/build-lint-test.yml